src/domains/simple.rs

```diff
@@ -118,10 +118,10 @@ impl Domain for SimpleVal {
     }
 
 
-}
```

Comment:
`git checkout main src/domains/simple.rs` will remove all changes to a file. This is useful in cases where you made changes without intending to